projects/plugins/jetpack/modules/gravatar-hovercards.php

```diff
@@ -200,19 +200,24 @@ function grofiles_get_avatar( $avatar, $author ) {
 				$cache_key   = 'gravatar_profile_' . $email_hash;
 
 				$response_body = wp_cache_get( $cache_key, $cache_group );
-				if ( false === $response_body ) {
-					$response = wp_remote_get( esc_url_raw( 'https://en.gravatar.com/' . $email_hash . '.json' ) );
-
+				if ( $response_body ) {
+					$profile      = $response_body->entry[0];
+					$display_name = $profile->displayName; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
+				} else {
+					$response = wp_remote_get( esc_url_raw( 'https://www.gravatar.com/' . $email_hash . '.json' ) );
 					if ( is_array( $response ) && ! is_wp_error( $response ) ) {
 						$response_body = json_decode( $response['body'] );
+						$profile       = $response_body->entry[0];
```

Comment:
It looks like if the email-hash does not exist, Gravatar returns a response like
```json
"User not found"
```
It might be more straightforward to go back to the old code and just do like this instead
```php
$profile      = isset( $response_body->entry[0] ) ? $response_body->entry[0] : null;
$display_name = isset( $profile->displayName ) ? $profile->displayName : ''; // phpcs:ignore 
```

Reply:
Thanks Brad! Logic fixed in https://github.com/Automattic/jetpack/pull/27828/commits/6ec7241f85185ffae47012c67b0ecbca9eaa68bf